selfservice/strategy/password/validator.go

```diff
@@ -62,12 +62,21 @@ type validatorDependencies interface {
 	config.Provider
 }
 
-func NewDefaultPasswordValidatorStrategy(reg validatorDependencies) *DefaultPasswordValidator {
+func NewDefaultPasswordValidatorStrategy(reg validatorDependencies) (*DefaultPasswordValidator, error) {
+	cache, err := ristretto.NewCache(&ristretto.Config{
+		NumCounters: 10 * 10000,
+		MaxCost:     60 * 10000, // BCrypt hash size is 60 bytes
+		BufferItems: 64,
```

Comment:
The cost in ristretto is set is kinda weird. For example, it is probably required to ignore the internal cost and use a custom cost function:

https://github.com/dgraph-io/ristretto/blob/8e850b710d6df0383c375ec6a7beae4ce48fc8d5/cache.go#L137

Reply:
Alright. I've added `ignoreInternalCost=true` but I'm not sure if custom cost function is needed at this point.